tests/modules/transformer/self_attention_test.py

```diff
@@ -1,20 +1,84 @@
 import copy
 import torch
+import pytest
 
 from allennlp.common import Params
+from allennlp.common import cached_transformers
+from allennlp.common.testing import assert_equal_parameters
+
 from allennlp.modules.transformer import SelfAttention
 from allennlp.common.testing import AllenNlpTestCase
 
+from transformers.configuration_bert import BertConfig
+from transformers.modeling_bert import BertSelfAttention
+from transformers.configuration_roberta import RobertaConfig
+from transformers.modeling_roberta import RobertaSelfAttention
+from transformers.configuration_electra import ElectraConfig
+from transformers.modeling_electra import ElectraSelfAttention
+from transformers.configuration_distilbert import DistilBertConfig
+from transformers.modeling_distilbert import MultiHeadSelfAttention
+
+# from transformers.configuration_mobilebert import MobileBertConfig
+# from transformers.modeling_mobilebert import MobileBertSelfAttention
+# from transformers.configuration_t5 import T5Config
+# from transformers.modeling_t5 import T5LayerSelfAttention
```

Comment:
What are these for?

Reply:
I wanted to test with different versions, but T5 requires some more changes. I am planning to add tests for them once I finish the first pass for all modules (i.e. all modules should at least generate the same output as vanilla bert).